### PR TITLE
Changing  Microsoft.Dotnet.GitHub.Authentication to  Microsoft.DotNet.GitHub.Authentication;

### PR DESCRIPTION
**Pull request by epananth**
Issue -> https://github.com/dotnet/core-eng/issues/9625

### CODE REVIEW

File: src/DotNet.Status.Web/InMemoryCacheInstallationLookup.cs [hunk 1]
```diff
@@ -7,8 +7,6 @@
 using System.Collections.Immutable;
 using System.Threading;
 using System.Threading.Tasks;
-using GitHubJwt;
```
Comment (epananth) [inline src/DotNet.Status.Web/InMemoryCacheInstallationLookup.cs:10]: This is no longer required by that file, so removed it.

**Comment by garath** [comment]
Just a note... this may confuse existing clones of the repo on case insensitive filesystems. It may require folks to do a bit of work to get the rename as intended.

Also I definitely think this should be done. /highfive

**Comment by riarenas** [comment]
>Just a note... this may confuse existing clones of the repo on case insensitive filesystems. It may require folks to do a bit of work to get the rename as intended.

I think there are no renames here (which makes it even more obvious that we should normalize to the DotNet version.

**Comment by riarenas** [comment]
Aaaand I just saw the edit. Wish GitHub would notify about those :|

**Pull request merged**